cip-00XX/cip-00XX.md

```diff
@@ -0,0 +1,146 @@
+- A new choice `DsoRules_CreateUnallocatedUnclaimedActivityRecord` allows the DSO to propose a beneficiary, reward amount, and reason for the reward.
+- Upon successful vote confirmation, an `UnallocatedUnclaimedActivityRecord` contract is created. This contract has two expiry parameters:
+  - `expiresAt`: Duration from creation, derived from the configurable DSO field `unallocatedUnclaimedActivityRecordTimeout` (defaulting to 24 hours).
+  - `unclaimedActivityRecordExpiresAt`: Passed to the resulting activity record once allocation occurs.
+
+### Allocation and Expiration
+
+- A new SV-App trigger, `UnallocatedUnclaimedActivityRecordAllocatorTrigger`, locates eligible `UnclaimedReward` contracts, archives them up to the specified amount, and creates an `UnclaimedActivityRecord`.
+- A second trigger, `UnallocatedUnclaimedActivityRecordExpiryTrigger`, expires unallocated records after their TTL.
+- A third trigger, `UnclaimedActivityRecordExpiryTrigger`, expires unused `UnclaimedActivityRecord` contracts.
+
+### Minting Mechanism
+
+- `UnclaimedActivityRecord` is introduced as a new transferable on-ledger asset.
+- It is added as a new variant (`InputUnclaimedActivityRecord`) within the existing `TransferInput` type.
+- This input is consumed by `AmuletRules_Transfer`, which mints the corresponding Amulet during a self-transfer executed by automation in the Wallet App (`CollectRewardsAndMergeAmuletsTrigger`).
+
+### Off-ledger Implementation Requirements
+
+- **SV App**
+  - Implement and deploy the three new triggers listed above.
+  - Extend the Treasury Service to support the new activity record lifecycle.
+
+- **Wallet App**
+  - Extend the `CollectRewardsAndMergeAmuletsTrigger` to auto-collect `UnclaimedActivityRecord` using the new transfer input.
+
+- **CLI App**
+  - Introduce a Python CLI utility to calculate minted and unclaimed rewards by party ID and time range, reusing logic from `scan_txlog.py`.
+
+- **SV App Voting UI**
+  - Update the SV App voting UI to support proposals using `DsoRules_CreateUnallocatedUnclaimedActivityRecord`.
+
+- **SV App Governance UI**
+  - Add support in the SV App for configuring `unallocatedUnclaimedActivityRecordTimeout`.
+
+## Motivation
+
+This change is required to implement the reward distribution mechanism described in CIP-0058. 
+The CIP defines a process whereby, upon successful milestone verification, the Tokenomics Working Group approves a 
+reward amount and the Super Validators assign a portion of unclaimed rewards to be minted. 
+This functionality leverages the standard voting process and introduces new ledger, off-ledger, and UI components to 
+support reward calculation, archival of unclaimed rewards, and minting of the corresponding Amulet.
+
+This aligns with the first deliverable of [CIP-0058](https://github.com/global-synchronizer-foundation/cips/blob/main/cip-0058/cip-0058.md): _Enable escrow mechanisms to ensure Super Validators deliver value in return for rewards_.
```

Comment:
I think it's worth mentioning that the proposed mechanism is more general and allows the SVs to mint any kind of unclaimed reward for any reason not just escrowed super validator rewards.